### PR TITLE
Enhancement: Set Deadline prio pool limit machine limit plugin per task type

### PR DESCRIPTION
**Pull request by friquette**
## Changelog Description
Create new settings for priority, pools, machine limit and plugin limit filtered by task type and host, replacing these in the host submitters settings. Also add these options in the publisher for Maya and Nuke.

## Testing notes:
1. set a profile in the settings
2. create an instance that can be sent to Deadline in Maya or Nuke
3. verify that the data in the publisher are correct and publish
4. verify that the data in Deadline are correct too

### CODE REVIEW

File: openpype/modules/deadline/maya/publish/submit_maya_deadline.py [hunk 1]
```diff
@@ -796,7 +800,7 @@ def get_attribute_defs(cls):
                       maximum=1000),
             TextDef("machineList",
                     label="Machine List",
-                    default="",
+                    default=cls.group,
```
Comment (BenSouchet) [inline openpype/modules/deadline/maya/publish/submit_maya_deadline.py:803]: This seems to be an error

**Comment by BenSouchet** [comment]
@friquette @ccaillot It seems ready to merge but since it's a pretty big PR can we all tests it one last time, and then indicate here in the comments if it's okay or not.

**Comment by friquette** [comment]
It seems to work fine with maya and nuke

**Review by ccaillot — APPROVED** [review]
It works for me

**Pull request merged**